includes/admin/customers/customers.php

```diff
@@ -313,9 +313,9 @@ function edd_customers_view( $customer ) {
 		</ul>
 	</div>
 
-	<?php do_action( 'edd_customer_before_purchases', $customer ); ?>
+	<?php do_action( 'edd_customer_before_downloads', $customer ); ?>
 
-	<div id="customer-purchases-wrapper" class="customer-section">
+	<div id="customer-downloads-wrapper" class="customer-section">
```

Comment:
This two lines should't have changed, this is the purchases table..not downloads. You've got these id's and actions repeated below:
https://github.com/easydigitaldownloads/Easy-Digital-Downloads/pull/3208/files#diff-f3394595b75bf8108fc776d387b357a1R360